### DISCUSSION TITLE
How to Create Subsite in Wordpress Multisite using LudicrousDB

### DISCUSSION THREAD

**Discussion by JoseLWal** [stuttter/ludicrousdb#148] [Q&A]
I am testing the plugin on a multisite environment with no other plugin installed, and if I try to create a subsite, the request is not  executed, not even if I am only connecting to the main database. I will really appreciate some help. Do I need to add a code that tells the plugin how to create the subsite? what is the code? Some help please.

**Comment by spacedmonkey** [comment]
Can you provide more information here? I am not sure what this post means? Is your install of multisite not creating tables? Are you expecting this plugin to automatically create new database per site? 

This plugin is a frame to allow you to route database calls to different databases and servers. This is logic you create yourself. It will. It do it automatically.
